### ISSUE TITLE
Release arrow-rs / parquet minor version `53.1.0` (October 2024)

### ISSUE THREAD

**Issue by alamb** [apache/arrow-rs#6340]
## Is your feature request related to a problem or challenge? Please describe what you are trying to do.

https://github.com/apache/arrow-rs/issues/6016 tracks releasing `53.0.0`

Our release schedule is here  https://github.com/apache/arrow-rs?tab=readme-ov-file#release-versioning-and-schedule

## Describe the solution you'd like
Checklist:
- [x] Release notes and bump version: https://github.com/apache/arrow-rs/pull/6501
- [x] Vote thread: https://lists.apache.org/thread/vk78wxzzky1hr77t8pnzhf1vs3wyh7hl
- [ ] Approved and deploy to crates.io

## Describe alternatives you've considered

According to the release schedule, the next release should be minor `53.1.0` (no API changes) released early October 2024

## Additional context
Suggested content:
TBD

**Comment by alamb** [comment]
I am hoping we can get https://github.com/apache/arrow-rs/pull/6303 into this release too

**Comment by alamb** [comment]
https://github.com/apache/arrow-rs/pull/6303 is merged. 

I have also tested upgrading DataFusion to this release in https://github.com/apache/datafusion/pull/12724 and things are looking good, so I will now proceed to completing the changelog and version https://github.com/apache/arrow-rs/pull/6501 and making a release candidate

**Comment by alamb** [comment]
I have created an RC and started a vote thread: https://lists.apache.org/thread/vk78wxzzky1hr77t8pnzhf1vs3wyh7hl

**Comment by kranurag7** [comment]
Hey thanks for cutting a new release, wondering if update to crates.io is manual here? Looking at versions info (https://crates.io/crates/arrow-json/versions) looks like it's manual. 

Is there something I can read on how long it takes the crate to be updated after a release is cut on GitHub? 

arrow-json crate on crates.io (https://crates.io/crates/arrow-json) still shows the version to be 53.0.0

**Comment by alamb** [comment]
> Is there something I can read on how long it takes the crate to be updated after a release is cut on GitHub?
> 
> arrow-json crate on crates.io (https://crates.io/crates/arrow-json) still shows the version to be 53.0.0

The release instructions are here: https://github.com/apache/arrow-rs/blob/master/dev/release/README.md

TLDR is there is a mandatory 3 day waiting period between creating the candidate and its final approval. When I am doing the release I typically will publish the release to crates.io as soon as I can after the release is officially approved

I have just completed the release process and the release is now available on crates.io

**Comment by alamb** [comment]
The various versions are now available on crates.io: https://crates.io/crates/arrow/53.1.0

Result thread: https://lists.apache.org/thread/z70xpdckbsqkcyrkw30gf5s2y3py3x56
